OWNERS

```diff
@@ -2,21 +2,20 @@
 # approval == this is a good idea /approve
 approvers:
   - kevinrizza
-  - benluddy
-  - awgreene
-  - dinhxuanvu
   - perdasilva
   - grokspawn
   - joelanford
   - tmshort
+  - dtfranz
+  - anik120
+
```

Comment:
Putting @dtfranz and myself as approvers, to share the load with other reviewers. I realized we don't have a formal process set up to discuss this thing, if anyone disagrees please feel free to call it out here or reach out in private and I shall revert this change.

Also @dtfranz hope you're okay with this 😅

Reply:
Yes, thanks for doing that :smile: